samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1660,21 +1655,18 @@ mfxStatus CDecodingPipeline::RunDecoding()
     bool                bErrIncompatibleVideoParams = false;
     CTimeInterval<>     decodeTimer(m_bIsCompleteFrame);
     time_t start_time = time(0);
-    MSDKThread * pDeliverThread = NULL;
 #if (MFX_VERSION >= 1025)
     mfxExtDecodeErrorReport *pDecodeErrorReport = NULL;
 #endif
 
-    if (m_eWorkMode == MODE_RENDERING) {
+    if (m_eWorkMode == MODE_RENDERING)
```

Comment:
please, don't change what you actually did not modify.

Reply:
fixed